### PR TITLE
Vultr: Introducing vr_os_facts module

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

This commit introduces a new module called vr_os_facts.

This module aims to return the list of OSes avaiable avaiable to use to
boot servers.

Sample available here:

```
"vultr_os_facts": [
  {
    "arch": "i386",
    "family": "ubuntu",
    "id": 216,    
    "name": "Ubuntu 16.04 i386",
    "windows": false
  }
]
```

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

  - vr_os_facts

##### ANSIBLE VERSION

  - devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -520,6 +520,7 @@ lib/ansible/modules/cloud/vultr/vr_dns_record.py E326
 lib/ansible/modules/cloud/vultr/vr_firewall_group.py E324
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E323
 lib/ansible/modules/cloud/vultr/vr_firewall_rule.py E324
+lib/ansible/modules/cloud/vultr/vr_os_facts.py E324
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt]: New ignores should not be added. Instead, the underlying issue should be fixed.

If the issue is in existing common code (docs fragments or module_utils), then the fix should be in a separate PR.

Comment (Spredzy) [reply test/sanity/validate-modules/ignore.txt]: @mattclay agree. I did that while https://github.com/ansible/ansible/pull/42464 gets merged, so the CI feedback is green. Once merged, I'll update this PR removing this line from here.

**Comment by Spredzy** [comment]
@resmo could I have your feedback on this one and it its ok with you? This way I can apply the same pattern for #42471 and #42470 . Thanks.

**Review by resmo — APPROVED** [review]
shipit
